### PR TITLE
Add new delimiter "+" for TargetQueues

### PR DESCRIPTION
**Pull request by smile21prc**
Fix to address https://github.com/dotnet/core-eng/issues/596.

### CODE REVIEW

**Comment by smile21prc** [comment]
This should work, however can't test it before it's merged.  @mmitche, can you review it?

**Comment by mmitche** [comment]
LGTM

**Comment by smile21prc** [comment]
Thanks Matt for the review. Will merge the PR when all checks are complete.

File: src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets [hunk 1]
```diff
@@ -117,11 +117,12 @@
...
 
     <!-- Split up Target Queues list 
-         In order to support both commas and semicolons, I do a little hoop-jumping to do a replace first
-         Note that queues will never have ',' or ';' in their name, and this greatly simplifies providing a queue list in *nix.
+         In order to support all delimiters (commas, plus and semicolons), I do a little hoop-jumping to do a replace first
+         Note that queues will never have ',', '+' or ';' in their name, and this greatly simplifies providing a queue list in *nix.
     -->
     <PropertyGroup>
       <ProcessedTargetQueues>$([System.String]::Copy('$(TargetQueues)').Replace(',',';'))</ProcessedTargetQueues>
+      <ProcessedTargetQueues>$([System.String]::Copy('$(TargetQueues)').Replace('+',';'))</ProcessedTargetQueues>
```
Comment (weshaggard) [inline src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets]: Have you verified this actually works from the bash shell and it doesn't try to interpret it in any other way?

**Pull request merged**